packages/Pill/src/Pill.styles.js

```diff
@@ -0,0 +1,76 @@
+import { css } from "styled-components";
+import stylers from "@paprika/stylers";
+import tokens from "@paprika/tokens";
+
+const pillColorStyles = {
+  "#d7d7d7": css`
+    color: ${tokens.color.black};
+
+    svg {
+      fill: ${tokens.color.black};
+    }
+  `,
+
+  noRisk: css`
+    background: ${tokens.color.blackLighten60};
+    color: ${tokens.color.black};
+  `,
+
+  lowRisk: css`
+    background: #299a7a;
+  `,
+
+  mediumRisk: css`
+    background: #c9af28;
+  `,
+
+  highRisk: css`
+    background: #cd3c44;
+  `,
+};
+
+const pillSizeStyles = {
+  small: css`
+    border-radius: ${tokens.space} / 50%;
+    font-weight: bold;
+    line-height: ${stylers.spacer(2)};
+    padding: 0 ${tokens.space};
+    ${stylers.fontSize(-3)};
+
+    .aclui-icon::before {
```

Comment:
☹️ Do we really need this? Can we just expect any `acl-ui` icons that are used to have their own css applied by the app?